plugin-server/src/worker/vm/lazy.ts

```diff
@@ -157,7 +157,6 @@ export class LazyPluginVM {
                         await this._setupPlugin(vm.vm)
                         this.ready = true
                     }
-                    status.info('🔌', `Loaded ${logInfo}.`)
```

Comment:
let's downgrade to `status.debug` instead? I'm currently debugging why geoip does not correctly start on hobby, and could use that info.